src/dmd/cli.d

```diff
@@ -756,7 +756,7 @@ dmd -cov -unittest myprog.d
         Feature("intpromote", "fix16997",
             "fix integral promotions for unary + - ~ operators"),
         Feature("dtorfields", "dtorFields",
-            "destruct fields of partially constructed objects"),
+            "destruct fields of partially constructed objects", false, false),
```

Comment:
You'll have to update `test/compilable/previewhelp.d` as it's now no longer documented.

Reply:
Definitly should have remembered that...